src/aoe_cut.jl

```diff
@@ -242,5 +356,32 @@ function get_continuum_survival_fraction(aoe::Vector{<:Unitful.RealOrRealQuantit
     return result, report
 end
 
+# Overloaded function for psd_cut only
+function get_continuum_survival_fraction(psd_parameter::Vector{<:Unitful.RealOrRealQuantity}, e::Vector{<:T}, center::T, window::T, psd_cut::Unitful.RealOrRealQuantity; 
+    inverted_mode::Bool=false, sigma_high_sided::Unitful.RealOrRealQuantity=Inf*unit(first(psd_parameter))) where T<:Unitful.Energy{<:Real}
+
+    cut_vector = if !inverted_mode
+        psd_cut .< psd_parameter .< sigma_high_sided
+    else
+        psd_parameter .< psd_cut
+    end
+    get_continuum_survival_fraction(cut_vector, psd_parameter, e, center, window, psd_cut; 
+                                    sigma_high_sided=sigma_high_sided)
+end
+
+# Overloaded function for psd_cut and existing cut_vector
+function get_continuum_survival_fraction(psd_parameter::Vector{<:Unitful.RealOrRealQuantity}, e::Vector{<:T}, center::T, window::T, psd_cut::Unitful.RealOrRealQuantity, cut_vector::BitVector; 
+    inverted_mode::Bool=false, sigma_high_sided::Unitful.RealOrRealQuantity=Inf*unit(first(psd_parameter))) where T<:Unitful.Energy{<:Real}
+
+    combined_cut_vector = if !inverted_mode
+        psd_cut .< psd_parameter .< sigma_high_sided .&& cut_vector
+    else
+        psd_parameter .< psd_cut .&& cut_vector
+    end
+
+    get_continuum_survival_fraction(combined_cut_vector, psd_parameter, e, center, window, psd_cut; 
+                                    sigma_high_sided=sigma_high_sided)
+end
```

Comment:
Again, `cut_vector` could obtain a default value of `trues(length(psd_parameter))` to avoid having two functions.

Reply:
Same things as discussed in https://github.com/legend-exp/LegendSpecFits.jl/pull/107#discussion_r1956241337